js/src/components/google-combo-account-card/conversion-measurement-notice/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppNotice from '.~/components/app-notice';
+
+const ConversionMeasurementNotice = () => {
```

Comment:
@ankitrox Since we just need to render the `AppNotice` component, let's render it directly in `ConnectedAccountsActions` instead of having a new `ConversionMeasurementNotice` component.

Reply:
@asvinb IMO, I think we still keep `ConnectedAccountsActions`  and `ConversionMeasurementNotice`  components separate as `ConnectedAccountsActions`  component will wrap the business logic for all components that may need (or may not need) to show depending on certain conditions and then the components themselves (like `ConversionMeasurementNotice`) should only be UI focused giving them better decoupled behaviour and testability.

CC: @joemcgill

Reply:
@asvinb After our discussion, I have removed `ConversionMeasurementNotice` component and used `AppNotice` directly within `ConnectedAccountsActions`.